--- a/speedtest-web/Makefile
+++ b/speedtest-web/Makefile
@@ -41,10 +41,14 @@
-config SPEEDTEST_WEB_COMPRESS_UPX
-	bool "Compress executable files with UPX"
-	default y
-endef
+config SPEEDTEST_WEB_COMPRESS_UPX
+	bool "Compress executable files with UPX"
+	default n
+endef
@@ -61,9 +65,6 @@
 define Build/Compile
 	$(call GoPackage/Build/Compile)
-ifeq ($(CONFIG_SPEEDTEST_WEB_COMPRESS_UPX),y)
-	$(STAGING_DIR_HOST)/bin/upx --lzma --best $(GO_PKG_BUILD_BIN_DIR)/speedtest
-endif
 endef
@@ -46,7 +50,9 @@
-	export GOPROXY=https://goproxy.baidu.com
+	export GOPROXY=
